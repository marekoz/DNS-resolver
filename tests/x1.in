8.8.8.8 -r -s kazi.fit.vutbr.cz -x