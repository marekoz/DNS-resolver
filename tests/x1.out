Authoritative: No, Recursive: Yes, Truncated: No
Question section (1)
  8.8.8.8.in-addr.arpa., Type: PTR, Class: IN
Answer section (1)
  8.8.8.8.in-addr.arpa., Type: PTR, Class: IN, TTL: <number>, Data length: 12, dns.google.
Authority section (0)
Additional section (0)